x-pack/plugins/security_solution/public/common/components/event_details/table/investigate_in_timeline_button.tsx

```diff
@@ -37,15 +39,28 @@ export const InvestigateInTimelineButton: React.FunctionComponent<{
     getDataViewsSelector(state)
   );
 
+  const hasTemplateProviders =
+    dataProviders && dataProviders.find((provider) => provider.type === 'template');
+
   const clearTimeline = useCreateTimeline({
     timelineId: TimelineId.active,
-    timelineType: TimelineType.default,
+    timelineType: hasTemplateProviders ? TimelineType.template : TimelineType.default,
   });
 
-  const configureAndOpenTimeline = React.useCallback(() => {
+  const configureAndOpenTimeline = useCallback(() => {
     if (dataProviders || filters) {
       // Reset the current timeline
-      clearTimeline();
+      if (timeRange) {
+        clearTimeline({
+          timeRange: {
+            kind: 'absolute',
+            from: timeRange,
+            to: new Date().toISOString(),
```

Comment:
Is it always to the current time?

Reply:
ya agree this was weirdly done by me, forgot to go back and change it. updated now so that timerange is passed as a prop or not at all. For the now, I think oldest timestamp -> now is as inclusive as we can be, since a value has to be supplied, not sure what else we could use.